Project.toml

```
@@ -3,22 +3,21 @@ uuid = "6218d12a-5da1-5696-b52f-db25d2ecc6d1"
 version = "0.7.5"
 
 [deps]
-BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
 ColorTypes = "3da002f7-5984-5a60-b8a6-cbb66c0b333f"
 FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
 FixedPointNumbers = "53c48c17-4a7d-5ca2-90c5-79b7896eea93"
 ImageCore = "a09fc81d-aa75-5fe9-8630-4744c3626534"
+ImageMagick_jll = "c73af94c-d91f-53ed-93a7-00f77d67a9d7"
 InteractiveUtils = "b77e0a4c-d291-57a0-90e8-8db25a27a240"
 Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 
 [compat]
-julia = "1"
-BinaryProvider = "≥ 0.3.0"
 ColorTypes = "≥ 0.2.7"
 FixedPointNumbers = "≥ 0.3.0"
 ImageCore = ">= 0.7.0"
+julia = "≥ 1.3"
```

```suggestion
julia = "^1.3"
```
General consensus seems to be that you don't want to promise compatibility with Julia 2.x.

---

Doesn't this mean that it won't be installed on Julia 1.4?  Maybe instead we should use `julia = "^1"`?

---

Nope, `^1.3` means any `1.x >= 1.3`. See https://julialang.github.io/Pkg.jl/v1/compatibility/#Caret-specifiers-1

---

You learn something new every day!